<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWindow</class>
 <widget class="QMainWindow" name="LogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/folder-blue-git-icon.png</normaloff>:/icons/folder-blue-git-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="logView">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
      </widget>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menurepo">
    <property name="title">
     <string>&amp;Repo</string>
    </property>
    <addaction name="actionRepoOpen"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menurepo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAllBranches"/>
   <addaction name="actionDisplayTags"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionCommit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRepoOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreateBranch">
   <property name="text">
    <string>Create Branch</string>
   </property>
  </action>
  <action name="actionDeleteBranch">
   <property name="text">
    <string>DeleteBranch</string>
   </property>
  </action>
  <action name="actionAllBranches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All branches</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionDisplayTags">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display tags</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCommit">
   <property name="text">
    <string>Commit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
